packages/api-explorer/src/block-types/Content.jsx

```diff
@@ -124,7 +127,7 @@ Loop.defaultProps = {
 
 Content.propTypes = {
   body: PropTypes.string,
-  flags: PropTypes.shape({}),
+  flags: PropTypes.shape(),
```

Comment:
This resolve a warning?

Reply:
Yeah. I can't remember the specific wording, but essentially it seemed to be telling me that specifying a shape of an empty object actually sets an expectation that the passed value will have no keys.